### ISSUE TITLE
Audio recording no longer works

### ISSUE THREAD

**Issue by TheERaichu** [koush/vysor.io#821]
I was trying to perform a screen recording and when I check the recording it has no audio. My phone and PC output audio fine, but the recording does not. Everything was working fine like, a couple of days ago. I haven't changed any settings. The only thing I noticed changed was Vysor acting strange (Changed the nickname I gave my phone back to the default serial number).

I turned on audio recording, selected "start now", turned on screen recording, turned off screen recording and then turned off audio recording. 

 - Windows 10
 - Vysor Version 4.0.62

 - Samsung Galaxy A70
 - OS: Android 11
 - https://gist.github.com/8293b7dd927d7dd2e963fb84c6bdb4bb

**Additional context**

**Comment by koush** [comment]
The screen recording has never recorded audio, it has always been screen only. Thanks for the reminder though, I'll try to get this fixed this upcoming release.

https://github.com/koush/vysor.io/issues/533

**Comment by TheERaichu** [comment]
I thought I clarified it enough, but let me try again; When pressing the headphones icon to record the audio it doesn't work. The video ends up having no sound when I checked afterwards. This wasn't a problem until recently. It used to record fine.

**Comment by koush** [comment]
> I thought I clarified it enough, but let me try again; When pressing the headphones icon to record the audio it doesn't work. The video ends up having no sound when I checked afterwards. This wasn't a problem until recently. It used to record fine.

Hi, I understood you the first time. I am saying, the app has never ever recorded audio when recording video. It plays the audio back on your computer, but it does not include that audio as part of any screen recordings.

**Comment by TheERaichu** [comment]
What are you talking about? I've recorded a few different recordings with audio intact.

**Comment by koush** [comment]
Audio recording is supported on iOS, but not Android.

**Comment by koush** [comment]
I'm looking at the code right now, and the screen recording feature within vysor is video only.
